maintainers-guide.md

```diff
@@ -189,36 +189,52 @@ issues.
 
 ## Team labels
 
-* `team-Android`: Issues for Android team ([Open
-  issues](https://github.com/bazelbuild/bazel/labels/team-Android))
-* `team-Apple`: Issues for Apple team (also MacOS, iOS etc) ([Open
-  issues](https://github.com/bazelbuild/bazel/labels/team-Apple))
-* `team-Configurability`: Issues for Configurability team ([Open
-  issues](https://github.com/bazelbuild/bazel/labels/team-Configurability))
-* `team-EngProd`: Issues for EngProd team (also release, distribution) ([Open
-  issues](https://github.com/bazelbuild/bazel/labels/team-EngProd))
-* `team-Execution`: Issues for Execution phase team (local and remote execution,
-  sandbox) ([Open
-  issues](https://github.com/bazelbuild/bazel/labels/team-Execution))
+* `team-Android`: Issues for Android team
+  * [Open issues](https://github.com/bazelbuild/bazel/labels/team-Android)
```

Comment:
This works fine in general, but indentation in lists should be 4 spaces (and the first item should also be indented accordingly with 3 spaces after the star), like this:

```
*   First item.
    *   Second item.
```

Again, what you have renders correctly, but it quickly becomes confusing when you want to add more complex elements in the list item. For example, a verbatim code block indented only by six spaces (4 for the code block, and 2 for the list item) wouldn't render correctly.

I find it easier to reason about things by always using the "correct" indentation. It may also make this list easier to read in its source form. But up to you.

Reply:
TIL and done.